db/migrations/0002_actor_remove_genre_first_name_remove_genre_last_name.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.0.2 on 2024-03-05 04:28
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Actor',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('first_name', models.CharField(max_length=255)),
+                ('last_name', models.CharField(max_length=255)),
+            ],
+        ),
+        migrations.RemoveField(
+            model_name='genre',
+            name='first_name',
+        ),
+        migrations.RemoveField(
+            model_name='genre',
+            name='last_name',
+        ),
+    ]
```

Comment:
It is a good practice to include one migration per app per PR in Django projects since it helps to avoid `migrations` folder overload in the future. Represent all changes done to models in one migration file per app per PR, please